### DISCUSSION TITLE
Can I access the notes/body from a release inside a workflow?

### DISCUSSION THREAD

**Discussion by tonyarnold** [community/community#26953] [Actions]
<p>I’d like to capture and pass along the release notes that I write for my GitHub release. Is it possible to get to these from within a workflow step attached to a <code>release</code> workflow trigger?</p>

**Comment by Crow-EH** [comment, marked as answer]
<p>Hi <a class="mention" href="/tonyarnold">@tonyarnold</a>,</p>
<p>You should be able to get all the information you want from the github.event.release context.</p>
<p>The information you need should be github.event.release.body, but there’s also github.event.release.tag_name for the version, github.event.release.name for the title, etc.</p>
<p>The structure of the release object can be found in the GitHub API documentation : <a href="https://developer.github.com/v3/repos/releases/#get-a-single-release" rel="nofollow noopener">https://developer.github.com/v3/repos/releases/#get-a-single-release</a></p>
